app/routes/task_route.py

```diff
@@ -0,0 +1,139 @@
+from flask import Blueprint, make_response, request, jsonify
+from app.models.task import Task
+from app import db
+from datetime import date
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+import os
+
+# Blueprints
+task_bp = Blueprint("task_bp", __name__, url_prefix="/tasks")
+
+# Helper Functions
+def get_task_with_task_id(task_id):
+    return Task.query.get_or_404(task_id, description={"details": "Invalid data"})
+
+def post_slack_message(task):
+    slack_client = WebClient(token=os.environ["SLACK_BOT_TOKEN"])
+
+    try:
+        response = slack_client.chat_postMessage(channel="#task-notifications",
+                                text=f"Someone just completed the task {task.title}")
+
+    except SlackApiError as e:
+        assert e.response["error"]
```

Comment:
I think maybe you meant something like `abort` here.  `assert` is mostly for writing tests.

Instead of failing if Slack is down (or your token expired) I recommend just logging an error:
```suggestion
        print(f'Could not post to Slack: {e.response["error"]}')
```